varnish/metadata.csv

```diff
@@ -1,343 +1,343 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 varnish.accept_fail,gauge,,connection,,Accept failures. This metric is only provided by varnish 3.x.,0,varnish,accept fail
-varnish.backend_busy,rate,,connection,,Maximum number of connections to a given backend.,0,varnish,backend busy
-varnish.backend_conn,rate,,connection,,Successful connections to a given backend.,0,varnish,backend conn
-varnish.backend_fail,rate,,connection,,Failed connections for a given backend.,0,varnish,backend fail
-varnish.backend_recycle,rate,,connection,,Backend connections with keep-alive that are returned to the pool of connections.,0,varnish,backend recycle
-varnish.backend_req,rate,,request,,Backend requests.,0,varnish,backend req
-varnish.backend_retry,rate,,connection,,Backend connection retries.,0,varnish,backend retry
-varnish.backend_reuse,rate,,connection,,Recycled connections that has were reused.,0,varnish,backend reuse
-varnish.backend_toolate,rate,,connection,,Backend connections closed because they were idle too long.,0,varnish,backend toolate
-varnish.backend_unhealthy,rate,,connection,,Backend connections not tried because the backend was unhealthy.,0,varnish,backend unhealthy
+varnish.backend_busy,rate,,connection,second,Maximum number of connections to a given backend.,0,varnish,backend busy
```

Comment:
Let's update the rate type to gauge as well

Reply:
oh, I left that uncommited